src/data/finsupp/ne_locus.lean

```diff
@@ -88,35 +88,44 @@ by { ext, simpa only [mem_ne_locus] using hF.ne_iff }
 
 end ne_locus_and_maps
 
-section cancel_and_group
 variables [decidable_eq N]
 
-@[simp] lemma add_ne_locus_add_eq_left [add_left_cancel_monoid N] (f g h : α →₀ N) :
-  (f + g).ne_locus (f + h) = g.ne_locus h  :=
+@[simp] lemma ne_locus_add_left [add_left_cancel_monoid N] (f g h : α →₀ N) :
+  (f + g).ne_locus (f + h) = g.ne_locus h :=
 zip_with_ne_locus_eq_left _ _ _ _ add_right_injective
 
-@[simp] lemma add_ne_locus_add_eq_right [add_right_cancel_monoid N] (f g h : α →₀ N) :
-  (f + h).ne_locus (g + h) = f.ne_locus g  :=
+@[simp] lemma ne_locus_add_right [add_right_cancel_monoid N] (f g h : α →₀ N) :
+  (f + h).ne_locus (g + h) = f.ne_locus g :=
 zip_with_ne_locus_eq_right _ _ _ _ add_left_injective
 
-@[simp] lemma neg_ne_locus_neg [add_group N] (f g : α →₀ N) : (- f).ne_locus (- g) = f.ne_locus g :=
+section add_group
+variables [add_group N] (f f₁ f₂ g g₁ g₂ : α →₀ N)
+
+@[simp] lemma ne_locus_neg_neg : ne_locus (-f) (-g) = f.ne_locus g :=
 map_range_ne_locus_eq _ _ neg_zero neg_injective
 
-lemma ne_locus_neg [add_group N] (f g : α →₀ N) : (- f).ne_locus g = f.ne_locus (- g) :=
-by rw [← neg_ne_locus_neg _ _, neg_neg]
+lemma ne_locus_neg : ne_locus (-f) g = f.ne_locus (-g) := by rw [←ne_locus_neg_neg, neg_neg]
+
+lemma ne_locus_eq_support_sub (f g : α →₀ N) : f.ne_locus g = (f - g).support :=
```

Comment:
I'll remove this unnecessary `(f g : α →₀ N)` in my PR, and I plan to use dot notation for `ne_locus` everywhere. Do you have different opinions?

Reply:
The dot notation goes against usage of `dist`, but I don't really care. I'll remove the arguments.